src/compas_fab/backends/ros/client.py

```diff
@@ -151,6 +153,14 @@ def __enter__(self):
     def __exit__(self, *args):
         self.close()
 
+    @property
+    def ros_distro(self):
+        """Retrieves the ROS version to which the client is connected (eg. kinetic)"""
+        if not self._ros_distro:
+            self._ros_distro = Param(self, '/rosdistro').get(timeout=3)
```

Comment:
woah, the distro retrieval gets a 3 second timeout, but the default for ik is only 1?

Reply:
hehe.. good point! made them more coherent on https://github.com/compas-dev/compas_fab/pull/279/commits/f8192e9757e968b7b8714165232969b3d38150a1